tensorflow/core/util/use_cudnn.cc

```diff
@@ -54,6 +54,22 @@ bool CudnnUseFrontend() {
   return result;
 }
 
+bool CudnnUseRuntime() {
```

Comment:
Where is this function / envvar documented?

Reply:
I add some comments about why we need this env var. But I am not sure how to add it to some docs.

Reply:
OK, this is about runtime compilation, I see.  Should it be called CudnnUseRuntimeCompilation, and same for the envvar?  Or is "cudnn runtime" a thing that people will understand to mean "runtime compilation"?  Or, I see elsewhere you call it "RuntimeFusion", that works for me too...